backend/src/api/regions.ts

```diff
@@ -0,0 +1,18 @@
+import { connectToDatabase, Organization } from '../models';
```

Comment:
There's an unused import for Organization

Reply:
Resolved [6ca0ca7](https://github.com/cisagov/XFD/pull/526/commits/6ca0ca7e6d0ade414828e350dc12b4b9e505503c)